externals/llvm-external-projects/torch-mlir-dialects/include/torch-mlir-dialects/Dialect/Tcp/IR/TcpBase.td

```diff
@@ -88,8 +88,7 @@ class Tcp_UnaryElementwiseOp<string mnemonic, list<Trait> traits = []> :
     Tcp_Op<mnemonic, !listconcat(traits, [
         Pure,
         Elementwise,
-        SameOperandsAndResultShape,
-        SameOperandsAndResultElementType])> {
+        SameOperandsAndResultShape])> {
```

Comment:
Since all elementwise ops except `tcp.sqrt` adhere to the `SameOperandsAndResultElementType` trait, is it better to leave this as-is, and let `Tcp_SqrtOp` just be a `Tcp_Op` that uses a relaxed set of traits?

Reply:
It is still useful to be able to identify all the unary elementwise ops using the common base class type. 

If there are more such ops in future, we could add another base type that can be common to all of them.